pandas/io/packers.py

```diff
@@ -506,11 +526,17 @@ def decode(obj):
         dtype = dtype_for(obj[u'dtype'])
         pd_dtype = pandas_dtype(dtype)
         np_dtype = pandas_dtype(dtype).base
+
+        ctor_dtype = np_dtype
+        if is_categorical_dtype(pd_dtype):
+            # Series ctor doesn't take dtype with categorical
+            ctor_dtype = None
```

Comment:
@jreback - at this point `obj['data']` is already a `Categorical` (and dtype is category), which is why this was necessay?

```python
In [37]: pd.Series(pd.Categorical([1,2,3]), dtype='category')
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-37-87a1a3228158> in <module>()
----> 1 pd.Series(pd.Categorical([1,2,3]), dtype='category')

ValueError: cannot specify a dtype with a Categorical
```